app/scripts/lib/setupSentry.js

```diff
@@ -225,7 +225,7 @@ export const SENTRY_BACKGROUND_STATE = {
     useTokenDetection: true,
     useRequestQueue: true,
   },
-  SelectedNetworkController: { domains: true, perDomainNetwork: false },
+  SelectedNetworkController: { domains: false, perDomainNetwork: true },
```

Comment:
We do not want to include `domains` in our sentry reports since it is potentially sensitive and violates our goal to preserve privacy, but we do want to include the `perDomainNetwork` toggle since it is not sensitive and potentially useful for diagnosing bugs.